README.md

```diff
@@ -1,10 +1,12 @@
 # Vertex AI SDK Test Data
-
 The Vertex AI SDK Test Data includes mock responses from Gemini models for use in unit testing of the Vertex AI for Firebase SDKs.
 
 ## Usage
-
-The mock responses are planned to be used in the Vertex AI for Firebase SDKs in the following repositories:
-- [firebase-android-sdk](https://github.com/firebase/firebase-android-sdk/)
+The mock responses are currently being used to test the Vertex AI for Firebase SDKs in the following repositories:
 - [firebase-ios-sdk](https://github.com/firebase/firebase-ios-sdk/)
-- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
\ No newline at end of file
+- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
+
+## Versioning
```

Comment:
Is it 'will be tagged' or 'should be tagged'?

Reply:
I meant that it will be tagged by GitHub Actions. I added that part in the latest commit.